pkg/crc/cluster/cluster.go

```diff
@@ -223,3 +223,15 @@ func DeleteOpenshiftApiServerPods(oc oc.OcConfig) error {
 
 	return errors.RetryAfter(60, deleteOpenshiftApiserverPods, time.Second)
 }
+
+func CheckProxyIsPartOfAuthOperator(oc oc.OcConfig, proxy *network.ProxyConfig) (bool, error) {
```

Comment:
`AuthOperator`? this is checking `openshift-marketplace`? I would pass the operator name as an argument to that method, and call it `CheckProxySettingsForOperator(...)`, `OperatorHasProxy`, ... or something like this?

Reply:
Ah I was testing Auth before but yes I will pass the namespace and deployment of operator as argument.